pkg/client/cli/cmds_intercept.go

```diff
@@ -241,21 +259,20 @@ func intercept(cmd *cobra.Command, args interceptArgs) error {
 			return err
 		}
 		return cliutil.WithManager(ctx, func(ctx context.Context, managerClient manager.ManagerClient) error {
-			safeCmd := cliutil.NewSafeCobraCommand(cmd)
-			is := newInterceptState(ctx, safeCmd, args, connectorClient, managerClient, connInfo)
+			is := newInterceptState(ctx, safeCobraCommandImpl{cmd}, args, connectorClient, managerClient, connInfo)
 			return client.WithEnsuredState(ctx, is, false, func() error {
 				if args.dockerRun {
 					return is.runInDocker(ctx, is.cmd, args.cmdline)
 				}
-				return cliutil.Run(ctx, safeCmd, is.env, args.cmdline[0], args.cmdline[1:]...)
+				return proc.Run(ctx, args.cmdline[0], args.cmdline[1:], is.env)
```

Comment:
When I was splitting Thomas's commit in to many commits, I didn't notice that he flipped around where `env` goes in the argument list, so I accidentally dropped that.  I don't think I care to change it.

Reply:
that was actually me making the Run function take in `args ...string` instead of `args []string`